action.yaml

```diff
@@ -293,7 +293,7 @@ runs:
         if [ "${{ env.SONAR_HOST_URL }}" != "" ] && [ "${{ env.SONAR_PROJECT_KEY }}" != "" ]  && [ "${{ steps.all_icons.outputs.sonar_scan }}" != "x" ]; then
           if [ "${{ github.event_name }}" == "pull_request" ]; then
           sonar_result="<${{ env.SONAR_HOST_URL }}/dashboard?id=${{ env.SONAR_PROJECT_KEY }}&pullRequest=${{ github.event.pull_request.number }}|SonarQube>"
```

Comment:
Just being nitpicking but, please indent the code probably. `sonar_result` should not be at the same level as the the `if` and also in the `else` block.

Reply:
Indented the code properly.